poolakey/src/main/java/ir/cafebazaar/poolakey/security/Security.kt

```diff
@@ -0,0 +1,80 @@
+package ir.cafebazaar.poolakey.security
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.pm.PackageManager
+import android.content.pm.Signature
+import android.os.Build
+import ir.cafebazaar.poolakey.BuildConfig
+import ir.cafebazaar.poolakey.constant.Const.BAZAAR_PACKAGE_NAME
+import ir.cafebazaar.poolakey.getPackageInfo
+import java.io.ByteArrayInputStream
+import java.io.InputStream
+import java.security.PublicKey
+import java.security.cert.CertificateFactory
+import java.security.cert.X509Certificate
+import java.util.*
+
+object Security {
+
+    private const val bazaarCertificateHex: String = BuildConfig.BAZAAR_HASH
```

Comment:
What's the point of having this constant value when you can use `BuildConfig.BAZAAR_HASH` directly? If your concern is the length of the code, then you can import `BuildConfig.BAZAAR_HASH` and use `BAZAAR_HASH` directly.

Reply:
Done